android/src/main/java/com/swmansion/gesturehandler/core/StylusData.kt

```diff
@@ -0,0 +1,26 @@
+package com.swmansion.gesturehandler.core
+
+import com.facebook.react.bridge.Arguments
+import com.facebook.react.bridge.ReadableMap
+
+data class StylusData(
+  val tiltX: Double = 0.0,
+  val tiltY: Double = 0.0,
+  val altitudeAngle: Double = 0.0,
+  val azimuthAngle: Double = 0.0,
+  val pressure: Double = -1.0
+) {
+  fun toReadableMap(): ReadableMap {
+    val stylusDataObject = Arguments.createMap()
+
+    stylusDataObject.putDouble("tiltX", tiltX)
+    stylusDataObject.putDouble("tiltY", tiltY)
+    stylusDataObject.putDouble("altitudeAngle", altitudeAngle)
+    stylusDataObject.putDouble("azimuthAngle", azimuthAngle)
+    stylusDataObject.putDouble("pressure", pressure)
+
+    val readableStylusData: ReadableMap = stylusDataObject
+
+    return readableStylusData
+  }
```

Comment:
```suggestion
  fun toReadableMap(): ReadableMap = Arguments.createMap().apply {
    putDouble("tiltX", tiltX)
    putDouble("tiltY", tiltY)
    putDouble("altitudeAngle", altitudeAngle)
    putDouble("azimuthAngle", azimuthAngle)
    putDouble("pressure", pressure)
  }
```

Doesn't this work?

Reply:
It does! I'm not so used to Kotlin tricks, so thanks for pointing this out! 😄 

Done in [cace1c9](https://github.com/software-mansion/react-native-gesture-handler/pull/3111/commits/cace1c9da94fd6ef7f5c1cb695371154355fe682)